qcfractal/interface/models/rest_models.py

```diff
@@ -37,36 +46,45 @@ def register_model(name: str, rest: str, body: 'ProtoModel', response: 'ProtoMod
 
     """
 
+    if is_custom_queries:
+        models_dict = __custom_rest_models
+    else:
+        models_dict = __rest_models
+
     name = name.lower()
     rest = rest.upper()
 
-    if (name in __rest_models) and (rest in __rest_models[name]):
+    if (name in models_dict) and (rest in models_dict[name]):
         raise KeyError(f"Model name {name} already registered.")
 
-    if name not in __rest_models:
-        __rest_models[name] = {}
+    if name not in models_dict:
+        models_dict[name] = {}
 
-    __rest_models[name][rest] = (body, response)
+    models_dict[name][rest] = (body, response)
 
 
-def rest_model(name: str, rest: str) -> Tuple['ProtoModel', 'ProtoModel']:
+def rest_model(resource: str, rest: str, subresource: str=None) -> Tuple['ProtoModel', 'ProtoModel']:
     """Aquires a REST Model
 
     Parameters
     ----------
-    name : str
-        The REST endpoint name.
+    resource : str
+        The REST endpoint resource name.
     rest : str
-        The REST endpoint type.
-
+        The REST endpoint type: GET, POST, PUT, DELETE
+    subresource: str
+        A subresource under the main resource
     Returns
     -------
     Tuple['ProtoModel', 'ProtoModel']
         The (body, response) models of the REST request.
 
     """
     try:
-        return __rest_models[name.lower()][rest.upper()]
+        if resource.lower() in __custom_rest_models:
+            return __custom_rest_models[resource.lower()][subresource.upper()]
+        else:
+            return __rest_models[resource.lower()][rest.upper()]
     except KeyError:
         raise KeyError(f"REST Model {name.lower()}:{rest.upper()} could not be found.")
```

Comment:
The error needs to be expanded to include the subresource requests since this is what the user will see.